src/shaka_scheme/system/base/BigInteger.hpp

```diff
@@ -153,10 +163,15 @@ namespace shaka {
          */
         friend bool operator!=(const BigInteger& lhs, const BigInteger& rhs);
 
-        std::string get_value() const;
+        std::string get_str_value() const;
+        std::vector<std::uint64_t> get_value() const;

```

Comment:
You may want to be careful about producing an entire `std::vector` everytime we use this type. The type may want to instead be `const std::vector<std::utint64_y>&` if possible to prevent the cost of copying.